selfdrive/car/hyundai/values.py

```diff
@@ -606,7 +607,9 @@ class Buttons:
   "use_fca": set([CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_2021, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ, CAR.IONIQ_EV_2020, CAR.KONA_EV, CAR.KIA_FORTE, CAR.KIA_NIRO_EV, CAR.PALISADE, CAR.GENESIS_G70, CAR.KONA, CAR.SANTA_FE, CAR.KIA_SELTOS]),
 }
 
-EV_HYBRID = set([CAR.IONIQ_EV_2020, CAR.IONIQ_EV_LTD, CAR.IONIQ, CAR.KONA_EV, CAR.KIA_NIRO_EV])
+# Ensure that all cars in PHEV are also in EV_HYBRID
+EV_HYBRID = set([CAR.IONIQ_EV_2020, CAR.IONIQ_EV_LTD, CAR.KONA_EV, CAR.KIA_NIRO_EV, CAR.IONIQ, CAR.KIA_OPTIMA_H])
```

Comment:
Not sure why I added Optima Hybrid, or why it wasn't here before, I'll check the gas signal tomorrow for it

Reply:
Ah, it was in non_tested_routes, so it wasn't checking gasPressed at all. I checked a (incomplete, only a few rlogs uploaded) route from this car and the current gasPressed signal it's using is always 0!